README.md

```diff
@@ -30,22 +30,26 @@ established successfully
 
 ```bash
 % ./tcpgoon --help
-Usage of ./tcpgoon:
+tcpgoon tests concurrent connections towards a server listening on a TCP port
+
+Usage:
+  tcpgoon <host> <port> [flags]
```

Comment:
flags after <host> <port> looks super strange! Can we move to tcpgoon  [flags] <host> <port> ?

Reply:
I will have a look, but it doesn't look to weird to me :)

Reply:
I've changed in usage, but in the end it has no effect, you can put flags before or after arguments.